src/libs/OptionsListUtils.js

```diff
@@ -368,18 +376,37 @@ function getNewGroupOptions(
  * Build the options for the Sidebar a.k.a. LHN
  * @param {Object} reports
  * @param {Object} personalDetails
- * @param {Obejct} draftComments
+ * @param {Object} draftComments
  * @param {Number} activeReportID
+ * @param {String} priorityMode
  * @returns {Object}
  */
-function getSidebarOptions(reports, personalDetails, draftComments, activeReportID) {
+function getSidebarOptions(reports, personalDetails, draftComments, activeReportID, priorityMode) {
+    let sideBarOptions = {
+        prioritizePinnedReports: true,
+        hideReadReports: false,
+        sortByAlphaAsc: false,
+    };
+    switch (priorityMode) {
+        case 'default':
+            break;
+        case 'gsd':
+            sideBarOptions = {
+                prioritizePinnedReports: false,
+                hideReadReports: true,
+                sortByAlphaAsc: true,
+            };
+            break;
+        default:
+            break;
```

Comment:
This is maybe a premature generalisation, but I built it like that to allow easy extending to include more priority modes/option combinations.

Reply:
>This is maybe a premature generalisation

I'd say it is. Let's please just `check priorityMode === 'gsd'`.

Reply:
+1